mappings/net/minecraft/world/PersistentStateContainer.mapping

```diff
@@ -0,0 +1,8 @@
+CLASS bbx net/minecraft/world/PersistentStateContainer
+	METHOD a getNextAvailableIdForPersistentState (Lbsg;Ljava/lang/String;)I
```

Comment:
(note: bbx is an interface, the verbosity might be warranted)

Reply:
This interface is implemented by `World` and `IWorld`. Dunno if this degree of clarity is needed in a class with a ton of stuff (maybe something else can also supply available ids?)